Jenkinsfile

```diff
@@ -53,8 +54,8 @@ def job = {
             "master": "master"
     ]
 
-    if (!config.isReleaseJob) {
-        ciTool("ci-update-version ${env.WORKSPACE} kafka")
+    if (config.nanoVersion && !config.isReleaseJob) {
+        ciTool("ci-update-version ${env.WORKSPACE} kafka", config.isPrJob)
```

Comment:
is it always the case that `config.isPrJob == !config.isReleaseJob` ?  just from a clarity perspective here, it's a little unnerving/uncertain to use `isReleaseJob` in the conditional, but `isPrJob` in the call to `ciTool`.

Reply:
after offline conversation, the difference between `isReleaseJob` & `isPrJob` makes more sense.